tests/test_sorting.py

```diff
@@ -21,8 +21,40 @@ def test_mergesort_empty():
 
 
 def test_bubble_sort():
-    # Stub for basic bubble sort tests, see issue #9
-    pass
+    # Test even-sized list:
+    l = [3, 2, 1, 5, 4, 6]
+    res = lws.bubble_sort(l)
+    assert res == [1, 2, 3, 4, 5, 6]
```

Comment:
This list of tests is good.  One way one can write it more compact is for example `pytest.mark.parametrize()`.
That way the input and expected results are parameter, and the repetitive code is a bit less (not much for the simple test).  It also effectively splits it in multiple tests.